modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,91 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.4 - 2023-03-15
+
+=== Added
+* New `tree` component that can be used in dialog body panel.
+* renderUI property in the `Theme` type can now return a `Promise<RenderResult>` instead of `RenderResult`.
+* New `isEditable` API to `editor.selection` that returns true or false if the current selection is editable.
+* New `isEditable` API to `editor.dom` that returns true or false if the specified node is editable.
+* New `setText` and `setIcon` methods added to menu button and toolbar button apis.
```

Comment:
```suggestion
* New `setText` and `setIcon` methods added to menu button and toolbar button API.
```

Reply:
Made the change locally and pushed upstream.